parsec/core/gui/users_widget.py

```diff
@@ -170,7 +170,7 @@ async def _do_list_users_and_invitations(core):
 
 async def _do_cancel_invitation(core, token):
     try:
-        await core.delete_invitation(token=token)
+        await core.delete_invitation(token=token, reason=InvitationDeletedReason.CANCELLED)
```

Comment:
Funny thing is `core.delete_invitation` doesn't use the `reason` parameters ! https://github.com/Scille/parsec-cloud/blob/39866ad773c2333dc10b2cdb8a1ff8b7b553b099/parsec/core/logged_core.py#L206-L208

I think we should modify `core.delete_invitation` to use the `reason` param and to provide it with a default value of `InvitationDeletedReason.CANCELLED` (hence nothing to change in `_do_cancel_invitation`)

Reply:
fixed